### PR TITLE
Addes style base property + StatusDot custom color

### PR DESCRIPTION
**Pull request by piny4man**

### CODE REVIEW

File: src/components/StatusDot/index.tsx [hunk 1]
```diff
@@ -3,11 +3,11 @@ import { IBaseProps, StatusType } from '../../models'
...
 
 interface IProps extends IBaseProps {
-  type: StatusType
+  type?: StatusType
 }
 
-const StatusDot: React.FC<IProps> = ({ type }) => {
-  return <Styled className={ `label ${ type }` } />
+const StatusDot: React.FC<IProps> = ({ type, style }) => {
+  return <Styled style={ style } className={ `label ${ type }` } />
```
Comment (mfmontenegro) [inline src/components/StatusDot/index.tsx]: check type:
`${ type || '' }`

**Review by ghost — CHANGES_REQUESTED** [review]
just one comment

File: src/components/StatusDot/index.tsx (hunk 1 above)
Comment (alberss) [reply src/components/StatusDot/index.tsx]: Done!

**Review by ghost — APPROVED** [review]
Done

**Pull request merged**